dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -218,6 +218,7 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/metrics/prometheus.yaml",
 		"libexec/alluxio-config.sh",
 		"LICENSE",
+		"microbench/target/benchmarks.jar",
```

Comment:
i noticed in all the other jars that are copied by this script, there is a version substring in the jar name; why doesn't this one have it?

Reply:
It's the uber jar generated by the shade plugin, not produced by maven packager. The name came from the [template](https://github.com/openjdk/jmh/blob/dfb595b1f1d965ba1ecfa3f0e15c2b29fd8cfe10/jmh-archetypes/jmh-java-benchmark-archetype/src/main/resources/archetype-resources/pom.xml#L75-L78) of JMH. We should change that to be `alluxio-microbench-VERSION-executable.jar` or something like that in here https://github.com/Alluxio/alluxio/blob/08eab54fdc99a3dee3eca2ffd16a986dbd47c2f6/microbench/pom.xml#L32

Reply:
It is an arbitrary choice defined here: https://github.com/Alluxio/alluxio/blob/5c091c202c7b7bc033b00123b01c43ca12ced60a/microbench/pom.xml#L32
It could easily be made to include the version number.

Reply:
I changed it to `alluxio-microbench-VERSION.jar`.

Reply:
The jar produced by maven jar plugin has also this name. Should we name the uber jar differently?

Reply:
Maybe that jar can be used somewhere else, i'm not sure